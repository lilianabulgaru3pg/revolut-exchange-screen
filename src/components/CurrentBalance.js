import React from 'react';
import Typography from '@material-ui/core/Typography';
import PropTypes from 'prop-types';

const CurrentBalance = ({ balance, err = false }) => (
  <Typography variant="subtitle2" color={err ? 'error' : 'initial'}>
    Balance:
    {balance}
  </Typography>
);

CurrentBalance.propTypes = {
  balance: PropTypes.string.isRequired,
  err: PropTypes.bool.isRequired,
};

export default CurrentBalance;
