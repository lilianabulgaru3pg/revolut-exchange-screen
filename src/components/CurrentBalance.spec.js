import React from 'react';
import renderer from 'react-test-renderer';
import CurrentBalance from './CurrentBalance';

describe('CurrentBalance Component', () => {
  test('snapshot renders', () => {
    const component = renderer.create(
      <CurrentBalance balance=" 11.00 Â£" err={false} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
