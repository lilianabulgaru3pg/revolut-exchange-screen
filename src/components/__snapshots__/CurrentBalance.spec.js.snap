// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentBalance Component snapshot renders 1`] = `
<h6
  className="MuiTypography-root MuiTypography-subtitle2"
>
  Balance:
   11.00 Â£
</h6>
`;
